### DISCUSSION TITLE
API test in dev mode returns 401 Authorization Required even though the authorization works fine in maker mode

### DISCUSSION THREAD

**Discussion by nethaniasonya** [bahasa-ai/otomatis-guidance#27] [Ideas]
![Screen Shot 2021-08-19 at 12 35 49](https://user-images.githubusercontent.com/47688745/130013618-abaa567a-889d-453e-a2ef-fb71adfc7ac0.png)

**Comment by jundialwan** [comment]
Hi @nethaniasonya, is this issue already resolved?

**Comment by nethaniasonya** [comment]
solved kak👍
